documentation/src/main/asciidoc/userguide/chapters/statistics/Statistics.adoc

```diff
@@ -0,0 +1,159 @@
+[[statistics]]
+== Statistics
+:stat-sourcedir: ../../../../../../../hibernate-core/src/main/java/org/hibernate/stat
+
+Hibernate can gather all sorts of statistics which can help you get a better insight into what Hibernate does behind the scenes.
+
+By default, the statistics are not collected because this incurs an additional processing and memory overhead. To instruct Hibernate to start collecting statistics, you need to set the `hibernate.generate_statistics` configuration property to `true`:
+
+====
+[source,xml]
+----
+<property
+    name="hibernate.generate_statistics"
+    value="true"
+/>
+----
+====
+
+[[statistics-methods]]
+=== org.hibernate.stat.Statistics methods
+
+The Hibernate statistics are made available via the
+https://docs.jboss.org/hibernate/orm/{majorMinorVersion}/javadocs/org/hibernate/stat/Statistics.html[`Statistics`] interface which exposes the following methods:
+
+[[statistics-general]]
+==== General statistics methods
+
+`isStatisticsEnabled`:: Are statistics enabled?
+`setStatisticsEnabled(boolean b)`:: Enable statistics based on the provided parameter.
+`clear`:: Reset all statistics.
+`logSummary`:: Print a summary of the current statistics into the application log.
+`getStartTime`:: The milliseconds (JVM standard `currentTimeMillis()`) from which all statistics accessed since the initial creation of this Statistics instance or the last `clear()` method call.
```

Comment:
I don't quite understand this sentence.

Reply:
Changed to: 

> The milliseconds (JVM standard `currentTimeMillis()`) since the initial creation of this Statistics instance or the last time `clear()` was called.